app/inputs/GoogleCalendar/Input.test.ts

```diff
@@ -32,7 +32,7 @@ const defaultConfiguration: GoogleCalendarConfiguration = {
 
 describe('getWorklogs', () => {
     beforeEach(() => {
-        mockedGetEventsList.mockResolvedValue({ data: { items: [] } });
+        mockedGetEventsList.mockResolvedValue({ data: { items: [], eventTypes: ["default"] } });
```

Comment:
The mocked response for getEventsList includes eventTypes: ["default"], but this addition does not align with the actual Google Calendar API response structure. The eventTypes parameter is intended for use in the request, not the response. Adjust the test to reflect realistic API behavior, focusing on how eventTypes influences the fetched events rather than being part of the response.